ScanToShop/Managers/AlertManager.m

```diff
@@ -0,0 +1,49 @@
+//
+//  AlertManager.m
+//  ScanToShop
+//
+//  Created by Joey R. Hernandez Perez on 7/17/20.
+//  Copyright © 2020 joeyrhernandez1. All rights reserved.
+//
+
+#import "AlertManager.h"
+
+@interface AlertManager ()
+
+@end
+
+@implementation AlertManager
+
++ (void)loginAlert:(NSString *)errorString ViewController:(UIViewController *)vc {
+    UIAlertController *alert;
+   
+    UIAlertAction *cancelAction = [UIAlertAction actionWithTitle:@"Cancel"
+                                                           style:UIAlertActionStyleCancel
+                                                         handler:^(UIAlertAction * _Nonnull action) {
+        NSLog(@"Cancel action performed");
+    }];
+    UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"OK"
+                                                       style:UIAlertActionStyleDefault
+                                                     handler:^(UIAlertAction * _Nonnull action) {
+        NSLog(@"OK action performed.");
+    }];
+    
+    if ([errorString isEqual:@""]) {
```

Comment:
This is kind of a weird way to do this. Consider using an enum for error types so you can just call the method with 'errorType:LoginErrorMissingInput' or something like that.